src/archana/SeleniumAss2.java

```diff
@@ -0,0 +1,64 @@
+package archana;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SeleniumAss2 {
+	WebDriver driver;
+
+	void facebookLogin() throws InterruptedException {
+		System.setProperty("webdriver.chrome.driver", "./resources/windows/chromedriver.exe");
+		driver = new ChromeDriver();
+		driver.get("https://www.facebook.com/");
+		String facebookPageTitle = driver.getTitle();
+		if (facebookPageTitle.equals("Facebook � log in or sign up"))
+			System.out.println(" Facebook page opened sucessfully");
+		else
+			System.out.println(" Incorrect page opened");
+
+		driver.manage().window().maximize();
+
+		// enter email and Pass
+		driver.findElement(By.id("email")).sendKeys("9028553461");
+		driver.findElement(By.name("pass")).sendKeys("Harsh@1017");
+		driver.findElement(By.name("login")).click();
+		Thread.sleep(500);
+
+		// verify login
+		if (driver.getTitle().equals("Facebook"))
+			System.out.println("\n Login sucessfully");
+		else
+			System.out.println("\n Login Fail");
+	}
+
+	void navigateOperation() {
+		driver.navigate().to(" https://www.google.com/");
+		driver.navigate().back();
+		if (driver.getTitle().equals("(2) Facebook"))
```

Comment:
Do you know why title is coming as "(2) Facebook" , bez you have 2 unread notification. what if there will mutliple in future.. You should use contains method.

Reply:
code is updated as per your comment.